File: Java/libraries/recognizers-text-number-with-unit/src/main/java/com/microsoft/recognizers/text/numberwithunit/NumberWithUnitOptions.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.microsoft.recognizers.text.numberwithunit;

public enum NumberWithUnitOptions {
    None(0);

    private final int value;

    NumberWithUnitOptions(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }
}

File: .NET/Microsoft.Recognizers.Text.NumberWithUnit/NumberWithUnitOptions.cs
using System;

namespace Microsoft.Recognizers.Text.NumberWithUnit
{
    [Flags]
    public enum NumberWithUnitOptions
    {
        /// <summary>
        /// Represents None
        /// </summary>
        None = 0,

        /// <summary>
        /// NoProtoCache
        /// </summary>
        NoProtoCache = 16,

        /// <summary>
        /// EnableCompoundTypes, mode that extracts compound dimension units as single entities.
        /// </summary>
        EnableCompoundTypes = 2097152, // 2 ^21,

        /// <summary>
        /// EnablePreview
        /// </summary>
        EnablePreview = 8388608, // 2 ^23
    }
}
